src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMap.java

```diff
@@ -363,7 +363,7 @@ public static void setGeometryProperty(final Geometry geometry, final String key
                 // Trying to override an existing value - this shouldn't happen!
                 if (!Objects.equals(existingValue, value))
                 {
-                    logger.error(
+                    logger.debug(
                             "Trying to override existing '{}' key's value of '{}' with '{}' for geometry {}",
                             key, existingValue, value, geometry.toString());
```

Comment:
Or other option is this, since logger will do the toString anyway:
```suggestion
                            key, existingValue, value, geometry);
```